index.html

```diff
@@ -42,6 +42,7 @@ <h1 class="fist-section__title"><div>Hey there. I'm Glauber<br></div>
 <section class="second__section">
```

Comment:
Kindly consider using more descriptive class names. It will help you navigate through your own code later on and for others to understand it more easier

Reply:
Thank you for this tip, I changed the classes.